spec/takeaway_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'takeaway'
+
+describe Takeaway do
+	
+	menu = {fish_and_chips: 5.99, curry_and_rice: 7.99, wun_ton_soup: 5.99, sweet_and_sour_pork: 6.50, prawns_chop_suey: 5.50}
```

Comment:
Metrics/LineLength: Line is too long. [123/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)